src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -734,7 +734,7 @@
   "TagCannotBeDeletedWhileInUse": "Cannot be deleted while in use",
   "TagIsNotUsedAndCanBeDeleted": "Tag is not used and can be deleted",
   "Tags": "Tags",
-  "TagsHelpText": "Applies to movies with at least one matching tag",
+  "TagsHelpText": "Tags list items will be added with",
```

Comment:
We need a new translation key, as this is also used in other places in the UI where it's correct